extensions/ql-vscode/src/view/results/raw-results-table.tsx

```diff
@@ -19,158 +18,157 @@ import { onNavigation } from "./results";
 import { tryGetResolvableLocation } from "../../common/bqrs-utils";
 import { ScrollIntoViewHelper } from "./scroll-into-view-helper";
 import { sendTelemetry } from "../common/telemetry";
+import { assertNever } from "../../common/helpers-pure";
 
-export type RawTableProps = ResultTableProps & {
+export type RawTableProps = {
```

Comment:
I'm not sure why we were requiring `ResultTableProps` as props but then barely accessing any of the members. I've changed this to just require `databaseUri`. The place where we call `RawTable` we do `{...this.props}` and this still works since unknown props are just ignored.